diff --git a/dist/src/index.d.ts b/dist/src/index.d.ts
index a92fe2d6fbc1044aa5100b7afa82e1c379282a43..eb1dc7e90253edf6e4a07ad1906f80c7a20578d3 100644
--- a/dist/src/index.d.ts
+++ b/dist/src/index.d.ts
@@ -4,9 +4,7 @@ import { ComponentMetadata } from './types/metadata';
 import { ReactivityType, TargetBlock } from './types/mitosis-component';
 export * from './flow';
 export * from './generators/alpine';
-export * from './generators/angular';
 export * from './generators/builder';
-export * from './generators/context/angular';
 export * from './generators/context/qwik';
 export * from './generators/context/react';
 export * from './generators/context/rsc';
@@ -35,7 +33,6 @@ export * from './helpers/is-mitosis-node';
 export { checkShouldOutputTypeScript } from './helpers/output';
 export * from './helpers/signals';
 export * from './helpers/typescript-project';
-export * from './parsers/angular';
 export * from './parsers/builder';
 export * from './parsers/context';
 export * from './parsers/jsx';
diff --git a/dist/src/index.js b/dist/src/index.js
index 0fcb37974f6e9b72d77e8862ab53e80200272aad..cd718e70fadeae8322504a6fac367155570c0d9b 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -17,9 +17,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.checkShouldOutputTypeScript = exports.createMitosisNode = exports.renameImport = exports.renameComponentImport = exports.renameComponentFile = exports.getComponentFileExtensionForTarget = exports.checkIsSvelteComponentFilePath = exports.checkIsMitosisComponentFilePath = exports.checkIsLiteComponentFilePath = void 0;
 __exportStar(require("./flow"), exports);
 __exportStar(require("./generators/alpine"), exports);
-__exportStar(require("./generators/angular"), exports);
 __exportStar(require("./generators/builder"), exports);
-__exportStar(require("./generators/context/angular"), exports);
 __exportStar(require("./generators/context/qwik"), exports);
 __exportStar(require("./generators/context/react"), exports);
 __exportStar(require("./generators/context/rsc"), exports);
@@ -57,7 +55,6 @@ var output_1 = require("./helpers/output");
 Object.defineProperty(exports, "checkShouldOutputTypeScript", { enumerable: true, get: function () { return output_1.checkShouldOutputTypeScript; } });
 __exportStar(require("./helpers/signals"), exports);
 __exportStar(require("./helpers/typescript-project"), exports);
-__exportStar(require("./parsers/angular"), exports);
 __exportStar(require("./parsers/builder"), exports);
 __exportStar(require("./parsers/context"), exports);
 __exportStar(require("./parsers/jsx"), exports);
diff --git a/dist/src/targets.d.ts b/dist/src/targets.d.ts
index 3a83b289da3cd90e2c84a4fc445c828c01c3f095..8ba63699d89e5afb5919dc6d15ba1d38550cfb50 100644
--- a/dist/src/targets.d.ts
+++ b/dist/src/targets.d.ts
@@ -1,7 +1,6 @@
 export declare const builder: (options?: import("./generators/builder").ToBuilderOptions) => ({ component }: import(".").TranspilerArgs) => import("@builder.io/sdk").BuilderContent;
 export declare const targets: {
     readonly alpine: import(".").TranspilerGenerator<import("./generators/alpine").ToAlpineOptions>;
-    readonly angular: import(".").TranspilerGenerator<import("./generators/angular").ToAngularOptions>;
     readonly customElement: import(".").TranspilerGenerator<import("./generators/html").ToHtmlOptions>;
     readonly html: import(".").TranspilerGenerator<import("./generators/html").ToHtmlOptions>;
     readonly mitosis: import(".").TranspilerGenerator<Partial<import("./generators/mitosis").ToMitosisOptions>>;
diff --git a/dist/src/targets.js b/dist/src/targets.js
index bc81124a3b94210edab2047469c16102caeda6ba..4e8e16b4bb2704a98b6955bf4c44f2b695519ebf 100644
--- a/dist/src/targets.js
+++ b/dist/src/targets.js
@@ -2,7 +2,6 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.targets = exports.builder = void 0;
 const alpine_1 = require("./generators/alpine");
-const angular_1 = require("./generators/angular");
 const builder_1 = require("./generators/builder");
 const html_1 = require("./generators/html");
 const liquid_1 = require("./generators/liquid");
@@ -23,7 +22,6 @@ const vue_1 = require("./generators/vue");
 exports.builder = builder_1.componentToBuilder;
 exports.targets = {
     alpine: alpine_1.componentToAlpine,
-    angular: angular_1.componentToAngular,
     customElement: html_1.componentToCustomElement,
     html: html_1.componentToHtml,
     mitosis: mitosis_1.componentToMitosis,
diff --git a/package.json b/package.json
index 0d4fe2a60990a1675d04918566c91d21e412a4ed..a4531f21ba5df501a6ed0d27f33083423d21f733 100644
--- a/package.json
+++ b/package.json
@@ -54,7 +54,6 @@
     "!**/*.tsbuildinfo"
   ],
   "dependencies": {
-    "@angular/compiler": "^11.2.11",
     "@babel/core": "7.14.5",
     "@babel/generator": "^7.14.3",
     "@babel/plugin-syntax-decorators": "^7.12.1",
@@ -73,8 +72,8 @@
     "prettier": ">=2.0.0 <3.0.0",
     "prettier-plugin-organize-imports": "^3.2.3",
     "prettier-plugin-svelte": "^2.10.0",
-    "svelte": "^3.30.0",
-    "svelte-preprocess": "^5.0.3",
+    "svelte": "5.55.7",
+    "svelte-preprocess": "6.0.3",
     "ts-morph": "^19.0.0",
     "typescript": "^5.3.2"
   },
